docs/next/en-US/state/global.md

```diff
@@ -24,4 +24,4 @@ Note that the global state has its own version of the page state store, and is c
 
 ## Pitfalls of request-time global state
 
-Absolutely critically, if you are using only request-time global state, and you use the `.get_global_state()` method *anywhere* in your app that uses build state, you will experience panics at build-time. This is because Perseus does not attempt to prevent pages from accessing global state at build-time, even if it won't exist until request-time: policing this is your responsibility. Further, even if you set sensible defaults at build-time and override these at request-time, any pages that only use build state will never see the request-time state until the client-side, which can lead to hydration errors. In general, be cautious when using request-time global state, as improper usage of it abounds, and it can be highly error-prone. If in doubt, avoid it.
+Absolutely critically, if you are using only request-time global state, and you use the `.get_global_state()` method *anywhere* in your app that uses build state, you will experience panics at build-time. This is because Perseus does not attempt to prevent pages from accessing global state at build-time, even if it doesn't exist until request-time: policing this is your responsibility. Further, even if you set sensible defaults at build-time and override these at request-time, any pages that only use build state will never see the request-time state until the client-side, which can lead to hydration errors. In general, be cautious when using request-time global state, as improper usage of it abounds, and it can be highly error-prone. If in doubt, avoid it.
```

Comment:
```suggestion
Absolutely critically, if you are using only request-time global state, and you use the `.get_global_state()` method *anywhere* in your app that uses build state, you will experience panics at build-time. This is because Perseus does not attempt to prevent pages from accessing global state at build-time, even if it won't exist until request-time: policing this is your responsibility. Further, even if you set sensible defaults at build-time and override these at request-time, any pages that only use build state will never see the request-time state until the client-side, which can lead to hydration errors. In general, be cautious when using request-time global state, as improper usage of it abounds, and it can be highly error-prone. If in doubt, avoid it.
```

I prefer "won't" here, personally.